docs/assets/diagrams/transfer/02-create-transaction.puml

```diff
@@ -0,0 +1,132 @@
+@startuml
+
+title Create Transaction
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+Alice -> App: I want to make this transaction
+
+rnote right of App #Light 
+  ""createDoc(""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    payee: {""
+  ""      phoneNumber: "+1-222-222-2222"""
+  ""    },""
+  ""    status: "PENDING"""
+  ""  }""
+  "")""
+end rnote
+
+App -> Firebase: Create a new document in the database
+return ""200 OK""
+
+rnote right of Firebase #Light
+  ""event(""
+  ""  "document.create",""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    payee: {""
+  ""      phoneNumber: "+1-222-222-2222"""
+  ""    },""
+  ""    status: "PENDING"""
+  ""  }""
+  "")""
+end rnote
+
+Firebase -> Server: Notify event for new document
+return ""200 OK""
+
+Server -> Adapter: ""**GET /parties/{Type}/{ID}**""
+return ""**HTTP 202** (Accepted)""
+
+Adapter -> Adapter: Lookup data
+
+rnote left of Adapter #Light
+  ""**PUT /parties/{Type}/{ID}**""
+  ""{ ""
+  ""  "party": {""
+  ""    "partyIdInfo": {""
+  ""      "partyIdType": enum,""
+  ""      "partyIdentifier": string,""
+  ""      "partySubIdOrType": string,""
+  ""      "fspId": number""
+  ""    },""
+  ""    "merchantClassificationCode": string,""
+  ""    "name": string,""
+  ""    "personalInfo": {""
+  ""      "complexName": {""
+  ""        "firstName": string,""
```

Comment:
I think the app displays this? Put in some sample data, I believe it is "Alice"?